html.go

```diff
@@ -49,7 +49,7 @@ type Renderer interface {
 type Gomponent = Renderer
 
 // HTML is a marker type used to differentiate between a string response and an HTML response.
-// To use templating, use [Ctx.Render].
+// To use templating, use [ctx.Render].
```

Comment:
```suggestion
// To use templating, use [ctx.Render].
```
You changed back here, but you renamed it to Ctx, no?